src/components/InventoryDetail/ApplicationDetails.js

```diff
@@ -14,14 +14,11 @@ const ApplicationDetails = ({ onTabSelect, appList, ...props }) => {
     const history = useHistory();
     const dispatch = useDispatch();
     const searchParams = new URLSearchParams(search);
-    const items = useSelector(({ entityDetails }) => entityDetails?.activeApps || [])
+    const items = useSelector(({ entityDetails }) => entityDetails?.activeApps || appList)
```

Comment:
The issues, I wrote as a comment can be solved by following, this also removes activeApps in redux completely:  
```suggestion
    const items =  (appList || [] )
```

Reply:
It'll be better to use safe accessor when selecting active app. But good point!